pyproject.toml

```diff
@@ -55,6 +55,7 @@ pre-commit = "^2.18.1"
 sphinxext-opengraph = "^0.6.3"
 slotscheck = "^0.14.0"
 sphinx-inline-tabs = "^2022.1.2-beta.11"
+pyright = "1.1.292"
```

Comment:
This breaks things on my system. Is there a possibility to only add it in CI?

Reply:
> Use a python Pyright wrapper so people do not need to install node for a Python project.

Reply:
The wrapper will install node for them anyways.

Reply:
So what? That is done automatically. this should be merged quite quickly since pyright regresses and fixes issues all the time every version. Runs like [this](https://github.com/nextsnake/nextcore/actions/runs/4224627971/jobs/7336565665) have completely unrelated errors.

Reply:
Id love to have this fixed quickly, however I want it done properly. Please change this.